### PR TITLE
Feature/<ptnam>

### PR DESCRIPTION
**Pull request by namdenn**

### CODE REVIEW

**Comment by tadkt** [comment]
You need to pull latest changes first then make pull request again

File: server.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        with open(filename, 'rb') as file:
+            chunk = file.read(1024)  
+            while chunk:
+                client_socket.send(chunk)
+                chunk = file.read(1024)
+    except FileNotFoundError:
+        error_message = f"Error: File '{filename}' not found."
+        print(error_message)  
+        client_socket.send(bytes(error_message, "utf-8"))  
+
```
Comment (tadkt) [inline server.py:16]: except one more:
except Exception as e:
  print(f"Error during file transfer: {e}")

File: server.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+s.listen(5)
+
+print("Server is listening...")
+
+while True:
+    clientsocket, address = s.accept()
+    print(f"Connection from {address} has been established!")
+
+    #Send file
+    sendfile(socket, "file.txt")
```
Comment (tadkt) [inline server.py]: I assume the arg "socket" is not making sense here. Either you change it or you notify @hoangha30 to change it. Remember to make the "Server setup" inside main() then call it at the end 👌

**Review by tadkt — COMMENTED** [review]
I commented 2 things in your code, review it once more 👯

File: server.py (hunk 2 above)
Comment (hoangha30) [reply server.py]: What do u mean about "not making sense"?

**Pull request merged**